distributed/worker.py

```diff
@@ -617,7 +617,7 @@ def __init__(
             "call_stack": self.get_call_stack,
             "profile": self.get_profile,
             "profile_metadata": self.get_profile_metadata,
-            "get_logs": self.get_logs,
+            "logs": self.logs,
```

Comment:
The `Worker` class seems to already have a `logs` property defined

https://github.com/dask/distributed/blob/ca88aa7327820fc934b2254b884721cea94991e5/distributed/worker.py#L731-L733

I haven't looked into the CI failures too deeply, but perhaps it's related

Reply:
Ah, that's unfortunate.  This change might not make sense I guess.  I suspect that a few folks  *do* use this, so we might want to go through a deprecation process if we do this, which might not be worth the effort.

Reply:
Ah I missed that!

They seem to have similar functionality, although the `get_logs` does a bit more checking and formatting.

I guess the main change would be going from a property to a method.